### ISSUE TITLE
Plz add option to change color of gurbani and bg color

### ISSUE THREAD

**Issue by SukhmanpreetSingh** [KhalisFoundation/sttm-desktop#6]

**Comment by navdeepsinghkhalsa** [comment]
@SukhmanpreetSingh This is a great suggestion, Singh Jee. It is currently planned and we are tracking it in a private JIRA instance, but we may be moving those issues to GH for the sake of transparency.

**Comment by sukhpalsingh** [comment]
I am thinking to have a 'Custom Theme' Option after the 'Dark Theme' in the settings. Once the user enable this option option, then show the controls for changing the background and font colors.

This way they can go back to the Dark / Light theme and then easily can switch to custom whenever they want.

**Comment by navdeepsinghkhalsa** [comment]
Absolutely. The custom options will be saved when toggling back between themes

**Comment by sukhpalsingh** [comment]
For the colors, are we going to use preset color classes e.g. heading-color-red, heading-color-blue etc.
or we provide ability to pick the color from the color picker.

I am more inclined to use the present color classes as it make it easier to set the boundary of the available colors that can be changed and make it easier for the user to find the right color more quickly.

**Comment by navdeepsinghkhalsa** [comment]
That's a good idea. Perhaps we can do both: presets and picker. I imagine that some users may desire to match colours with their "brand" like at a Gurdwara Sahib. What do you think?

**Comment by sukhpalsingh** [comment]
Looks good. i will start working on this.

**Comment by tarunsingh5** [comment]
Sukhpal Singh Ji - wanted to check if you made any progress on this

**Comment by sukhpalsingh** [comment]
I am working on this at the moment. I am trying to find a js library which we can use to pick the colors.

**Comment by tarunsingh5** [comment]
@sukhpalsingh I'm not a dev by any means but is this package usable? http://jscolor.com/

**Comment by sukhpalsingh** [comment]
it looks good. i also found another one https://bgrins.github.io/spectrum/
which i think is good to use as it is easy to manage.

**Comment by tarunsingh5** [comment]
i like it! - let's implement it :)

**Comment by tarunsingh5** [comment]
@sukhpalsingh any luck?

**Comment by sukhpalsingh** [comment]
working on it today. thanks

**Comment by tarunsingh5** [comment]
@sukhpalsingh any update here? can @inderpreetsingh help you with this?

**Comment by sukhpalsingh** [comment]
i have added jquery based color picker and i am being told to remove jquery which is a dependency for the color picker library. I order to proceed further, i will need to do research for library without jquery and can be install through npm and need to re-factor all to remove jquery calls. I will do in the following week if possible.

**Comment by navdeepsinghkhalsa** [comment]
I apologize for the lack of feedback on this, @sukhpalsingh. Your hard work is much appreciated. What do you think about jscolor.com? @tarunsingh5 mentioned it above as well

**Comment by sukhpalsingh** [comment]
Looks good but does not have npm install so i will include from the source i guess.

**Comment by tarunsingh5** [comment]
any update on this? @inderpreetsingh can you help maybe?

**Comment by sukhpalsingh** [comment]
happy if @inderpreetsingh take this one. sorry that i could not apply the fix in last week because of busy scheudle.

**Comment by inderpreetsingh** [comment]
Got this one! Started working on it

**Comment by sukhpalsingh** [comment]
i have redone all the changes in new branches as belows:
https://github.com/sukhpalsingh/sttm-core/commit/1ef4ef325f00ef83cc75a5ea5f7fd5f4c21a3dfd
https://github.com/sukhpalsingh/sttm-desktop/commits/feature/6-custom-theme-with-color-picker (last two commits)

You can use the changes as it is has color picker implementation done already. Only missing bit left is apply the color from preference to the custom theme classes. i.e. gurbani, teeka etc. I could not figure this out as we are using object for the display instead of html for the slide view now.
